test/Client.test.js

```diff
@@ -62,7 +71,7 @@ describe('Client code', () => {
           .waitForRequest((requestObj) => requestObj.url().match(/sockjs-node/))
```

Comment:
When the port is `9000`, `requestObj.url()` is `http://localhost:9000/main.js` and then `requestObj.url()` will be `http://localhost:9001/sockjs-node/info?t=xxxx` after this page will be proxyed.